native/core/src/common/bit.rs

```diff
@@ -671,7 +671,7 @@ impl BitReader {
             }
         }
 
-        debug_assert!((offset + i) % 8 != 0 || num_bits_to_read - i < 8);
+        debug_assert!((offset + i).is_multiple_of(8) || num_bits_to_read - i < 8);
```

Comment:
This is missing a `!`.

Reply:
Thanks, I ended up reverting this before seeing the comment. This was a suggestion from nightly Rust's clippy.